frappe/utils/fixtures.py

```diff
@@ -38,7 +38,7 @@ def import_fixtures(app):
 
 		file_path = frappe.get_app_path(app, "fixtures", fname)
 		try:
-			import_doc(file_path)
+			import_doc(file_path, sort=True)
```

Comment:
Where would you have this be configurable?
As far as I understand it, fixtures are mainly imported during migration, which calls sync_fixtures, which calls import_fixtures.
I don't think it should be configured in hooks.py, because the fixture import mechanism does not look at hooks.py at all as far as I can tell.
Should migrate get a new command line argument?